### ISSUE TITLE
vscodium on portableapps.com to easily manage its installation and updates

### ISSUE THREAD

**Issue by advait404** [VSCodium/vscodium#1295]
Can vscodium be published on the portableapps.com platform? It can be useful for easily managing portable installations of the app in a user-friendly manner.

**Comment by advait404** [comment]
No, sorry. I only use the platform as a user and don't know about app packaging. Maybe you can look into the [portableapps.com forums](https://portableapps.com/search/node/vscodium)

**Comment by Crystal-RainSlide** [comment]
PortableApps is somewhat distinct from common packaging solutions, also not that well-documented. I think it's best to invite someone who already know about creating PortableApps, to bootstrap this & describe the process of building a PortableApp, then @daiyam can add it to the GH Action workflows etc.

---

Related docs:

- https://portableapps.com/development
- https://code.visualstudio.com/docs/editor/portable

Existing projects for reference:

- https://github.com/Makazzz/VSCodiumPortable
- https://github.com/garethflowers/vscode-portable
- https://github.com/daemondevin/vscode-portable

---

For those who just want a more user-friendly portable solution, and don't care if it's a PortableApp or not:

- https://portapps.io/app/vscodium-portable/
- https://portapps.io/app/vscode-portable/

**Comment by gitgrub** [comment]
Wow maybe this is what I have been looking for.
I would like to have the IDE sitting in a single folder, without writing to the registry or user folder, and no interaction with external servers like microsoft or the like, you know just isolated, private.
Would you think that https://portapps.io/app/vscodium-portable/ is the best solution for this case?

Looking at this site and the formats to download, "Portable setup" is recommended but without reason.
Is the 7z file ok for a first install, or would I face troubles?
If the setup file is nice, that's ok for me for sure.

**Comment by advait404** [comment]
@gitgrub I do not think that  https://portapps.io/app/vscodium-portable/ supports auto updating, unlike portableapps.com 

However I have never used the platform, so I am not 100% sure about this. If auto - updating is not a concern, then simply using the portable version from github to download and update the app should be enough.

**Comment by gitgrub** [comment]
@advait404: I don't care about auto updating, I think updating every few months is ok, or if there is an actual problem for me.
For me its just about being clean an non intrusive.
Just not sure if installer or 7z is the better approach.

**Comment by gitgrub** [comment]
Ok will go for the zip, thank you sir.

**Comment by advait404** [comment]
@gitgrub I do not think portapps.io is needed. Its like a third party for portable apps, but I don't really see any advantages since VS codium already has a portable version.

Also, sorry for the confusion. You are right, you need the VSCodium-win32-x64-1.73.0.22306.zip file not the src.zip. 
I think you can download the win32 zip and try out the app. It will isolate all its files and user data and deleting it if something goes wrong should be easy.

This might be useful: https://code.visualstudio.com/docs/editor/portable

**Comment by advait404** [comment]
@gitgrub no problem, I totally get it. Its great to have apps that are fully portable so that I can easily restore them if something goes wrong. Its really useful for devices with low storage, and reduces risk of any complications that can happen from an install.

Vs codium is fully portable, based on my limited testing. I reend creating a 'workspace' folder somewhere to save all of your workpsace configurations. I use the extension 'workspace explorer' supplement do this. 

Also, I recommend using BC uninstaller Foss app for proper uninstalling of apps and all their system/user data.
